### ISSUE TITLE
add to cart error?

### ISSUE THREAD

**Issue by theewl** [Mosuswalks/snkrs-bot#9]
anyone able to successfully add a product to cart? i get a cors or server blocked error

**Comment by zer0awes0me** [comment]
@theewl I am also having the same issue. On the network it shows a 403 error. Were you able to find a work around?

**Comment by theewl** [comment]
@zer0awes0me nah.. they upgraded their security system so it’s preventing bots (Puppeteer) from accessing their site.

**Comment by igorthurow** [comment]
Hey guys, some workaround?

**Comment by zer0awes0me** [comment]
I was able to get it working on Indian website by using chrome normal and not chromium.
